UPGRADE-2.x.md

```diff
@@ -1,2 +1,9 @@
 UPGRADE 2.x
 ===========
+
+### GoogleAuthenticator
+
+ * Deprecated `GoogleAuthenticator::getUrl()` in favor of `GoogleQrUrl::generate`.
```

Comment:
headline: deprecated xYz and body, how someone could avoid the deprecate including the signature, I would say